kazoo/tests/test_watchers.py

```diff
@@ -418,6 +418,40 @@ def changed(children):
         update.wait(0.5)
         eq_(all_children, ['smith'])
 
+    def test_remove_session_watcher_on_func_stops(self):
```

Comment:
Since this test is concerning the ChildrenWatch watcher, maybe it is better to precise it in the test's name